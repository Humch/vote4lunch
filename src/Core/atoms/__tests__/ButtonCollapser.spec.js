/* eslint-disable no-undef */
import React from 'react';

import renderer from 'react-test-renderer';


import { ButtonCollapser } from '../..';


describe('ButtonCollapser Snapshot', () => {
  let collapse = false;
  function toggle() {
    collapse = !collapse;
  }

  it('capturing Snapshot of ButtonCollapser 0', () => {
    const renderedValue = renderer.create(
      <ButtonCollapser
        toggle={() => toggle()}
        collapse={collapse}
      />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });

  it('capturing Snapshot of ButtonCollapser 1', () => {
    toggle();
    const renderedValue = renderer.create(
      <ButtonCollapser
        toggle={() => toggle()}
        collapse={collapse}
      />
    ).toJSON();
    expect(renderedValue).toMatchSnapshot();
  });
});
