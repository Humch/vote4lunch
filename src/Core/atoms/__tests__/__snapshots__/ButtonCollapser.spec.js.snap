// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonCollapser Snapshot capturing Snapshot of ButtonCollapser 0 1`] = `
<button
  aria-label={null}
  className="mr-3 btn btn-info btn-sm"
  onClick={[Function]}
  type="button"
>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-angle-right fa-w-8 "
    data-icon="angle-right"
    data-prefix="fas"
    role="img"
    style={Object {}}
    viewBox="0 0 256 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</button>
`;

exports[`ButtonCollapser Snapshot capturing Snapshot of ButtonCollapser 1 1`] = `
<button
  aria-label={null}
  className="mr-3 btn btn-info btn-sm"
  onClick={[Function]}
  type="button"
>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-angle-down fa-w-10 "
    data-icon="angle-down"
    data-prefix="fas"
    role="img"
    style={Object {}}
    viewBox="0 0 320 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</button>
`;
